src/components/NavBar/styles.ts

```diff
@@ -0,0 +1,37 @@
+import { Theme } from '@mui/material/styles';
+import { makeStyles } from '@mui/styles';
+
+export default makeStyles((theme: Theme) => ({
+
+  toolbar: {
+    height: '80px',
+    width: '100%',
+    display: 'flex',
+    justifyContent: 'space-between',
+    // marginLeft: '480px',
```

Comment:
You don't need to leave in commented out code if you have git to do the "remembering" for you.